Gorilla/helpers.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"strconv"
+	"time"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func format_datetime(timestamp string) string {
+	// Convert the string to an int64 (timestamp in seconds)
+	unixTime, err := strconv.ParseInt(timestamp, 10, 64)
+	if err != nil {
+		return "format_datetime error"
+	}
+
+	// Create a time.Time object from the Unix timestamp
+	t := time.Unix(unixTime, 0)
+
+	// Format the time into the desired display format
+	return t.Format("2006-01-02 @ 15:04")
+}
+
+func isNil(i interface{}) bool {
```

Comment:
```suggestion
return i == nil || i == interface{}(nil)
```

Reply:
You killed the function signatute xD
But I get you, no need for if-else <3